src/main/webapp/js/instructorStudentListAjax.js

```diff
@@ -1,72 +1,25 @@
-    }
-    return str.replace(/</g, '&lt;')
-              .replace(/>/g, '&gt;')
-              .replace(/\'/g, '&quot;')
-              .replace(/\//g, '&#x2f;')
-              .replace(/\'/, '&#39;')
-              // To ensure when apply sanitizeForHtml for multiple times, the string's still fine
-              // Regex meaning: replace '&' with safe encoding, but not the one that is safe already
-              .replace(/&(?!(amp;)|(lt;)|(gt;)|(quot;)|(#x2f;)|(#39;))/g, '&amp;');
-}
-
-function sanitizeForJs(str) {
-    return sanitizeForHtml(str.replace(/\/\//g, "\\\\")
-                              .replace(/\"/g, "\\\"")
-                              .replace(/\'/g, "\\'")
-                              .replace(/#/g, "\\#"));
-}
-
-function getCourseStudentDetailsLink(student, userId) {
-    var link = '/page/instructorCourseStudentDetailsPage';
-    link = addParamToUrl(link, 'courseid', student.course);
-    link = addParamToUrl(link, 'studentemail', student.email);
-    link = addParamToUrl(link, 'user', userId);
-    return link;
-}
-
-function getCourseStudentEditLink(student, userId) {
-    var link = '/page/instructorCourseStudentDetailsEdit';
-    link = addParamToUrl(link,'courseid', student.course);
-    link = addParamToUrl(link, 'studentemail', student.email);
-    link = addParamToUrl(link, 'user', userId);
-    return link;
+function transportSectionChoices() {
+    var sectionChoices = $('.section-to-be-transported');
+    sectionChoices.remove();
+    $('#sectionChoices').append(sectionChoices);
+    sectionChoices.removeClass('section-to-be-transported');
 }
 
-function getCourseStudentDeleteLink(student, userId) {
-    var link = '/page/instructorCourseStudentDelete';
-    link = addParamToUrl(link, 'courseid', student.course);
-    link = addParamToUrl(link, 'studentemail', student.email);
-    link = addParamToUrl(link, 'user', userId);
-    return link;
+function transportTeamChoices() {
+    var teamChoices = $('.team-to-be-transported');
+    teamChoices.remove();
+    $('#teamChoices').append(teamChoices);
+    teamChoices.removeClass('team-to-be-transported');
 }
 
-function getStudentRecordsLink(student, userId) {
-    var link = '/page/instructorStudentRecordsPage';
-    link = addParamToUrl(link, 'courseid', student.course);
-    link = addParamToUrl(link, 'studentemail', student.email);
-    link = addParamToUrl(link, 'user', userId);
-    return link;
-}
-
-function addParamToUrl(url, key, value) {
-    if (typeof key == 'undefined' || typeof value == 'undefined'
-                                  || url.indexOf('?' + key + '=') != -1
-                                  || url.indexOf('&' + key + '=') != -1) {
-        return url;
-    }
-    url += url.indexOf('?') != -1 ? '&' : '?';
-    url += key + '=' + escape(value);
-    return url;
+function transportEmailChoices() {
+    var emailChoices = $('div[id^="student_email-c"]');
+    emailChoices.remove();
+    $('#emails').append(emailChoices);
```

Comment:
Yupp, then perhaps it shouldn't be done here, not worth the HTML changes but maybe we can open a new issue for this? Looks easier to understand, read and reason if all three functions do similar things since their names are really similar as well.